.env

```diff
@@ -1 +1,2 @@
-SKIP_PREFLIGHT_CHECK=true
\ No newline at end of file
+SKIP_PREFLIGHT_CHECK=true
+JWT_SECRET =G7u1K1rUPs
```

Comment:
This whole file is being tracked. Untrack this file from github and create a new secret key. This key is no longer secure.